bitnami/phpmyadmin/templates/ingress.yaml

```diff
@@ -1,5 +1,9 @@
 {{- if .Values.ingress.enabled -}}
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
+apiVersion: networking.k8s.io/v1beta1
+{{- else }}
 apiVersion: extensions/v1beta1
+{{- end }}
```

Comment:
Can we use a macro as we did for WordPress? Check the links below:

- https://github.com/bitnami/charts/blob/master/bitnami/wordpress/templates/ingress.yaml#L2
- https://github.com/bitnami/charts/blob/master/bitnami/wordpress/templates/_helpers.tpl#L183